introduction.md

```diff
@@ -0,0 +1 @@
+Hi I am Sevil. New to Git. Going through a Mastering Git class on CodeSchool.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.